app/controllers/admin/content/system-images.js

```diff
@@ -0,0 +1,33 @@
+import Ember from 'ember';
+import { eventTopics } from 'open-event-frontend/utils/dictionary/event';
+import { keys, uniqBy } from 'lodash';
+
+const { Controller, computed } = Ember;
+
+export default Controller.extend({
+  imageTopic: keys(eventTopics)[0],
+
+  topics: computed(function() {
+    return keys(eventTopics);
+  }),
+
+  imageObjects: computed('imageTopic', function() {
+    let subTopics = eventTopics[this.get('imageTopic')];
+    let filteredImageArray = this.get('model').map(function(obj) {
```

Comment:
uniqBy is used to remove redundant "Other" section. Since "Other" in many sections, it is displayed multiple times on a page. Infact nearly 19 times. Hence I used uniqBy.